com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -906,6 +918,107 @@ public bool StartHost()
             return IsListening;
         }
 
+#if RELAY_INTEGRATION_AVAILABLE
+// TODO: FIXME - Whether UNITY_WEBGL is defined or not, users might want to have a build that supports both modes.
```

Comment:
This area should be revisited and the connection type a setting (possibly add it to NetworkConfig?) should be exposed as a property in the inspector view. 
Unless cross play doesn't work with WEBGL...and/or could WSS be used in a non-web based runtime? (if the answer to both is no then let me know and I will remove the TODO comment).

Reply:
WebGL only works with wss (https://forum.unity.com/threads/how-to-use-ngo-with-webgl.1444894/#post-9058495), so for a WebGL build there is no choice. 

Cross play should work according to https://docs.unity3d.com/Packages/com.unity.transport@2.0/manual/cross-play.html

For desktop, this is where it becomes an important point of conversation indeed: how much configurability do we want to add here (also with regards to region choice, authentication, etc)? My preference is to keep it simple for the reasons I evoked [here](https://confluence.unity3d.com/display/MTT/Proof-of-Concept%3A+Integration+of+Relay+with+NetworkManager?focusedCommentId=343222078#comment-343222078), but I do not mind discussing it